sync/class.jetpack-sync-module-attachments.php

```diff
@@ -40,4 +42,19 @@ function send_attachment_info( $attachment_id ) {
 		}
 
 	}
+
+	function process_update( $attachment_id, $attachment_after, $attachment_before ) {
+		// Check whether attachment was added to a post for the first time
+		if ( 0 === $attachment_before->post_parent && 0 !== $attachment_after->post_parent ) {
+			/**
+			 * Fires when an existing attachment is added to a post for the first time
+			 *
+			 * @since 6.5.0
```

Comment:
Since 6.5 is out, you'll need to update this to 6.6.0 I'm afraid!

Reply:
No prob, no big rush on this one!